src/components/AccountList.tsx

```diff
@@ -17,7 +17,7 @@ function getDeployedContracts(account: Account): string {
   const contracts = account.deployedContracts.map(
     contract => contract.split(".").slice(-1)[0]
   );
-  return contracts.join(", ");
+  return "("+ contracts.length +") contracts ... "+ contracts.join(", ");
```

Comment:
Maybe we can render this as a list instead of simply a string, cause it will be cutout most of the time 🤔

Reply:
how would you want to display on the UI?

Reply:
It's better to display which contracts are currently deployed there than simply a number :)
Maybe `<ul><li/></ul>` block?
Or simply make `<p>` block with `<a>` links, which would open tab for that contract?

Reply:
Made the changes in this commit: https://github.com/onflow/flow-playground/pull/177/commits/1df5ce5050fc693c3f1fc0a9ef7109d3e25c32f2
But it looks a bit weird when there are more than 2 deployed contracts.

<img width="228" alt="Screen Shot 2021-11-21 at 7 09 55 PM" src="https://user-images.githubusercontent.com/81572453/142762182-66b7642d-a2c1-4292-8a7c-6feaffcc5c59.png">